Documentation/html_viewer/js/dictionary_functions.js

```diff
@@ -25,6 +30,29 @@
 		return tState.data;
 	}
 	
+
+	//customized sort for API dictionaries
+	function compareDictionaryObject(entryObject1,entryObject2) {
+		if(entryObject1["display name"] == "LiveCode Script" && entryObject2["display name"] == "LiveCode Builder")
+			return -1;
+		if(entryObject2["display name"] == "LiveCode Script" && entryObject1["display name"] == "LiveCode Builder")
+			return 1;
+		if(entryObject1["display name"] == "LiveCode Script")
+			return -3;
+		if(entryObject2["display name"] == "LiveCode Script")
+			return 3;
+		if(entryObject1["display name"] == "LiveCode Builder")
+			return -2;
+		if(entryObject2["display name"] == "LiveCode Builder")
+			return 2;
+		if(entryObject1["display name"].toLowerCase() < entryObject2["display name"].toLowerCase())
+			return -1;
+		if (entryObject1["display name"].toLowerCase() > entryObject2["display name"].toLowerCase())
+			return 1;
+		return 0;
+	}
+
```

Comment:
You don't need that many comparisons for LCS and LCB, it would be slightly more efficient as
```
		if(entryObject1["display name"] == "LiveCode Script"
			return -1;
		if(entryObject2["display name"] == "LiveCode Script"
			return 1;
		if(entryObject1["display name"] == "LiveCode Builder")
			return -1;
		if(entryObject2["display name"] == "LiveCode Builder")
			return 1;
...
```